### PR TITLE
Update transaction status content and handle error cases

### PR DESCRIPTION
**Pull request by thebeyondr**

### CODE REVIEW

File: components/report-details/support/transaction-status.tsx [hunk 1]
```diff
@@ -29,7 +29,10 @@ const TransactionStatus = ({
 			<div
 				className={cn("flex justify-center", {
 					"animate-spin":
-						status.label !== "Confirmed" && status.label !== "Failed",
+						status.label !== "Confirmed" &&
+						status.label !== "Failed" &&
+						status.label !== "InsufficientFunds" &&
+						status.label !== "ActionRejected",
```
Comment (baumstern) [inline components/report-details/support/transaction-status.tsx:35]:
```suggestion
						!(status.label === "Confirmed" ||
						status.label === "Failed" ||
						status.label === "InsufficientFunds" ||
						status.label === "ActionRejected"),
```
just I think it's cleaner...maybe :p

**Review by baumstern — APPROVED** [review]
thanks, LGTM! Just left one comment as a nitpick but feel free to ignore it

File: hooks/use-buy-fraction.ts [hunk 2]
```diff
@@ -56,10 +59,10 @@ const useHandleBuyFraction = (
 
       setTransactionStatus("SignForBuy");
       const myAmount = BigInt(order.price) * amount;
-      console.log(`myAmount: ${myAmount}`);
+      // console.log(`myAmount: ${myAmount}`);
       const tx = await call({ value: myAmount });
 
-      console.log(`amountInDollars: ${amountInDollars}`);
+      // console.log(`amountInDollars: ${amountInDollars}`);
```
Comment (CJ-Rose) [inline hooks/use-buy-fraction.ts:65]: done with console logs?
lines 43 44 62 65 85 90

File: components/reports/reports-filters.tsx [hunk 3]
```diff
@@ -223,25 +224,27 @@ export const FilterItems: React.FC<FilterItemsProps> = ({
...
+							</DrawerClose>
+						</div>
 					</section>
-					<DrawerClose>
-						<Button onClick={applyFilters} className="w-full">
-							Apply
-						</Button>
+					<DrawerClose onClick={applyFilters} className="w-full">
+						Apply
 					</DrawerClose>
```
Comment (CJ-Rose) [inline components/reports/reports-filters.tsx:248]: was blue button bg removed intentionally? imo was better with it.

current
Screenshot 2024-04-05 at 4 42 48 PM

previous
Screenshot 2024-04-05 at 4 43 22 PM

Comment (thebeyondr) [reply components/reports/reports-filters.tsx:248]: that was not intentional hmm, nice catch!

**Pull request merged**